packages/kit/src/cli.js

```diff
@@ -41,10 +41,6 @@ prog
 			return;
 		}
 
-		if (event === 'postinstall' && process.env.INIT_CWD) {
-			process.chdir(process.env.INIT_CWD);
-		}
-
```

Comment:
We don't need an analogue of this (from #5771) anymore?

Reply:
err.... yes. yes we do